scout/server/blueprints/clinvar/views.py

```diff
@@ -60,8 +60,7 @@ def clinvar_submissions(institute_id):
         "institute": institute_obj,
         "variant_header_fields": CLINVAR_HEADER,
         "casedata_header_fields": CASEDATA_HEADER,
-        "show_submit": institute_obj.get("clinvar_key")
-        and current_user.email in institute_obj.get("clinvar_submitters", []),
+        "show_submit": current_user.email in institute_obj.get("clinvar_submitters", []),
```

Comment:
This is still a slight drift in user experience compared to before the "saving api-keys" change. The user would have to be on the submitter list, even if there is no key set for the institute. It does make sense for the big institutes, but I would add a check if the submitters_list is populated, so that institutes can choose whether to have user list access control or just use paste the API key. Most of our power users are not admin, and only admins can set the key/add users.

Reply:
Agreed, I like freedom. Will fix!